src/main/groovy/org/reficio/p2/resolver/eclipse/impl/DefaultEclipseResolver.groovy

```diff
@@ -18,12 +18,12 @@
  */
 package org.reficio.p2.resolver.eclipse.impl
 
-import org.apache.commons.io.FilenameUtils
 import org.reficio.p2.logger.Logger
 import org.reficio.p2.resolver.eclipse.EclipseResolutionRequest
 import org.reficio.p2.resolver.eclipse.EclipseResolutionResponse
 import org.reficio.p2.resolver.eclipse.EclipseResolver
 
+@edu.umd.cs.findbugs.annotations.SuppressFBWarnings("SE_NO_SERIALVERSIONID")
```

Comment:
No import? You use imports in other classes.

Reply:
I think my idea fools me here. I will add it manually. Thanks